core_lib_generator/template_core_lib/setup.py

```diff
@@ -2,14 +2,19 @@
 
 import setuptools
 
-import core_lib
+# template_core_lib_import
 
 from setuptools import find_namespace_packages, setup
 from pip._internal.network.session import PipSession
 from pip._internal.req import parse_requirements
 
 dir_path = os.path.dirname(os.path.realpath(__file__))
 install_reqs = parse_requirements(os.path.join(dir_path, 'requirements.txt'), session=PipSession)
+requirements = []
+try:

```

Comment:
is this if statment needed now?

Reply:
it's for generating the list of requirements which was failing earlier